### DISCUSSION TITLE
Possible to paste only the next item each time Ctrl+V is pressed?

### DISCUSSION THREAD

**Discussion by Gitoffthelawn** [hluk/CopyQ#1693] [Q&A]
I have a hunch this isn't possible because CopyQ is just performing a paste operation, but I figure it never hurts to ask.

Let's say CopyQ currently contains 3 items:
```
1. alpha
2. beta
3. gamma
```

The first time I press Ctrl+V, I would like CopyQ to paste `alpha`.  So far, so good.

If I then immediately press Ctrl+V again (or it could be a different keyboard shortcut), I would like CopyQ to remove `alpha` and paste `beta`.

With another press of Ctrl+V (or the other keyboard shortcut, if needed), I would like CopyQ to remove `beta` and paste `gamma`.

Any chance of this being possible?

**Comment by hluk** [comment]
There might have been similar requests in the past. The closest things I could find:
- #597 (copy and paste from a special "Stack" tab)
- https://github.com/hluk/CopyQ/issues/729#issuecomment-323612936

**Reply by Gitoffthelawn** [reply]
Thank you.  The first link looks like it might be promising.  I'll definitely try it.

I also found https://github.com/hluk/CopyQ/issues/1447, which might be related... need to read it a couple more times to figure out what the OP is really trying to achieve.
